docs/source/getting-started.rst

```diff
@@ -103,6 +103,20 @@ Deploy
   python manage.py configure aws_account_id 111111111111  # Replace with your 12-digit AWS account ID
   python manage.py configure prefix <value>               # Choose a unique name prefix (alphanumeric characters only)
 
+.. note::
+
+  * Update the ``file_format`` value in ``conf/lambda.json``, choose ``parquet`` or ``json``. We will set default value to ``parquet`` in the future release.
+
+  .. code-block:: bash
+
+    "athena_partition_refresh_config": {
+      "concurrency_limit": 10,
+      "file_format": "parquet",
+      "log_level": "info"
+    }
+
+  * For more information, please visit :ref:`historical_search` page for more information.
```

Comment:
this is redundant.. `For more information .... for more information`

Please update to:

> More information can be found on the :ref:`historical_search` page

Reply:
actually -- can we stop using `ref` items so much. they're ugly an unnecessary

Reply:
can you update this to:

```
More information can be found on the `historical search <historical-search.html>`_ page.
```

see example here: https://raw.githubusercontent.com/airbnb/streamalert/530a154fd446a133868a49997bfbb47deb598c82/docs/source/datasources.rst:

```
To configure datasources, read `datasource configuration <conf-datasources.html>`_
```